core/src/service/usServiceListeners_p.h

```diff
@@ -60,6 +60,10 @@ class ServiceListeners : private MultiThreaded<>
 
 private:
 
+  std::mutex frameworkListenerMapMutex;
```

Comment:
Using threading related types directly will conflict with the threading option in CMake turned off. I suggest to use the same approach as `bundleListenerMap` to handle that transparently.